static-site/src/components/ListResources/useSortAndFilter.ts

```diff
@@ -49,13 +58,13 @@ export const useSortAndFilter = (sortMethod, selectedFilterOptions, originalData
 /* helper function to sort alphabetically. If provided with YYYY-MM-DD strings
  * this is the same as a chronological sort (oldest to newest)
  */
-function _lexicographicSort(a,b) {
+function _lexicographicSort(a: string, b: string) {
```

Comment:
```suggestion
function _lexicographicSort(a: string, b: string): 1 | -1 | 0 {
```

Reply:
Updated in [force-push](https://github.com/nextstrain/nextstrain.org/compare/5d85bd9d18b175c0d7a304960eccbce53e411f82..1ecff3bd305af2b9942c6d8d9b1283e6ff80a8a2).